### DISCUSSION TITLE
I have not the height infos (up/down) in the main screen

### DISCUSSION THREAD

**Discussion by andrearicci** [OpenTracksApp/OpenTracks#829] [Q&A]
Hi,
On the main screen I get distances, timing, but not the heights. Is it because I don't have an altimeter sensor? I tought it can be taken from the gps data. The value seems to be in the diegrams (by distance, by time) but is difficult to interpreter.
Xiaomi Redmi Note 4.
Thank you

**Comment by rgmf** [comment]
Yes, it's because you don't have a barometer.
There already is an issue open you can follow and it can interests you: https://github.com/OpenTracksApp/OpenTracks/issues/619
